massa-final-state/src/final_state.rs

```diff
@@ -0,0 +1,99 @@
+//! Copyright (c) 2022 MASSA LABS <info@massa.net>
+
+//! This file defines the final state of the node, which includes
+//! the final ledger and async message pool that are kept at
+//! the output of a given final slot (the latest executed final slot),
+//! and need to be bootstrapped by nodes joining the network.
+
+use crate::{
+    bootstrap::FinalStateBootstrap, config::FinalStateConfig, error::FinalStateError,
+    state_changes::StateChanges,
+};
+use massa_ledger::FinalLedger;
+use massa_models::Slot;
+use std::collections::VecDeque;
+
+/// Represents a final state (ledger, async pool)
```

Comment:
Is that normal that for the moment the `FinalState` has only a ledger and no async pool ?

Reply:
yes, it will be a followup, this is the first step for async integration. I didn't want to have to re-write documentation when it's done